### PR TITLE
android / iOS: german translations for v4.2

### PR DESCRIPTION
**Pull request by mlanp**
android / iOS: german translations added and fixed

### CODE REVIEW

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -712,38 +712,38 @@
...
-  <string name="snd_group_event_changed_member_role">***you changed role of %s to %s</string>
-  <string name="snd_group_event_changed_role_for_yourself">***you changed role for yourself to %s</string>
-  <string name="snd_group_event_member_deleted">sie haben <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> entfernt.</string>
-  <string name="snd_group_event_user_left">sie haben verlassen</string>
-  <string name="snd_group_event_group_profile_updated">gruppenprofil aktualisiert</string>
+  <string name="rcv_group_event_changed_member_role">änderte die Rolle von %s auf %s</string>
+  <string name="rcv_group_event_changed_your_role">änderte Ihre Rolle auf %s</string>
+  <string name="rcv_group_event_member_deleted">entfernte <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
+  <string name="rcv_group_event_user_deleted">hat Sie aus der Gruppe entfernt</string>
+  <string name="rcv_group_event_group_deleted">Gruppe gelöscht</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-de/strings.xml:722]: or does it necessarily have to be uppercase grammatically?
```suggestion
  <string name="rcv_group_event_group_deleted">gruppe gelöscht</string>
```

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -712,38 +712,38 @@
...
+  <string name="rcv_group_event_updated_group_profile">Aktualisiertes Gruppenprofil</string>
+  <string name="rcv_group_event_invited_via_your_group_link">wurde über Ihren Gruppen-Link eingeladen</string>
+  <string name="snd_group_event_changed_member_role">Sie haben die Rolle von %s auf %s geändert</string>
+  <string name="snd_group_event_changed_role_for_yourself">Sie haben Ihre eigene Rolle auf %s geändert</string>
+  <string name="snd_group_event_member_deleted">entfernt <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
+  <string name="snd_group_event_user_left">hat die Gruppe verlassen</string>
+  <string name="snd_group_event_group_profile_updated">Gruppenprofil aktualisiert</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-de/strings.xml:729]: same here
```suggestion
  <string name="rcv_group_event_updated_group_profile">aktualisiertes Gruppenprofil</string>
  <string name="rcv_group_event_invited_via_your_group_link">wurde über Ihren Gruppen-Link eingeladen</string>
  <string name="snd_group_event_changed_member_role">sie haben die Rolle von %s auf %s geändert</string>
  <string name="snd_group_event_changed_role_for_yourself">sie haben Ihre eigene Rolle auf %s geändert</string>
  <string name="snd_group_event_member_deleted">entfernt <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
  <string name="snd_group_event_user_left">hat die Gruppe verlassen</string>
  <string name="snd_group_event_group_profile_updated">gruppenprofil aktualisiert</string>
```

Comment (jr-simplex) [reply apps/android/app/src/main/res/values-de/strings.xml:729]: also maybe other words shouldn't be capitalized as well, e.g. "aktualisiertes gruppenprofil" - these service messages were intended to be as little intrusive as possible and have no accent

**Comment by mlanp** [comment]
Needs to be uppercase - grammar!

"Gruppe gelöscht"
"Aktualisiertes Gruppenprofil"
"Gruppenprofil aktualisiert"


-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 08:48 CEST, schrieb JRoberts ***@***.***>:

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -712,38 +712,38 @@
...
 
   <!-- GroupMemberRole -->
   <string name="group_member_role_member">Mitglied</string>
   <string name="group_member_role_admin">Admin</string>
   <string name="group_member_role_owner">Eigentümer</string>
 
   <!-- GroupMemberStatus -->
-  <string name="group_member_status_removed">entfernt</string>
-  <string name="group_member_status_left">verlassen</string>
+  <string name="group_member_status_removed">Entfernt</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-de/strings.xml:737]: same for member statuses

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -802,8 +802,8 @@
   <string name="error_changing_role">Fehler beim Ändern der Rolle</string>
   <string name="info_row_group">Gruppe</string>
   <string name="info_row_connection">Verbindung</string>
-  <string name="conn_level_desc_direct">direkt</string>
-  <string name="conn_level_desc_indirect">indirekt (<xliff:g id="conn_level">%1$s</xliff:g>)</string>
+  <string name="conn_level_desc_direct">Direkt</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-de/strings.xml:805]: same here

**Comment by mlanp** [comment]
Pls leave it is corrected by me. Grammar!

"Gruppenprofil aktualisiert"

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 08:50 CEST, schrieb JRoberts ***@***.***>:

**Comment by mlanp** [comment]
Pls leave it as it is - Grammar! This is not "intrusive".

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 08:52 CEST, schrieb JRoberts ***@***.***>:

**Comment by mlanp** [comment]
This is context sensitive. But I guess this is some kind of status information after the removement of the user which gives him the information that he is now removed from the group?
The same information if he is leaving the group for its own.

In this cases uppercase wording is ok.

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 08:54 CEST, schrieb JRoberts ***@***.***>:

**Comment by mlanp** [comment]
Ok, what's the context for this messages "direct" / "indirect"? When do you get this information?

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 08:54 CEST, schrieb JRoberts ***@***.***>:

**Comment by jr-simplex** [comment]
@mlanp member statuses displayed when you open the group info via its header (in top of a specific chat), and direct/indirect connection level are in member views (you can click on a member in group list), e.g. for connection level it just reads like this: "Connection                  direct"

**Comment by jr-simplex** [comment]
> Pls leave it as it is - Grammar! This is not "intrusive".

Didn't mean it as something bad really, just that these group event messages were not really considered to be proper sentences - just short informational member status updates, so they're fully lowercase except for occasional member profile name in them. So it might be okay if they're not grammatically correct in a sense for example that a sentence has to start with an upper case. I guess it's okay to leave as is if they become jarring in lowercase

**Comment by mlanp** [comment]
Ok, in this case it doesn't matter if it is uppercase or lowercase. Regarding my written change request in the next sentence, I would write it lowercase because than we are talking about some kind of sentence.
I would change the translation in front of this status. At the moment I see in the German translation "Verbindung steht". I would change it to "Verbindung besteht".

Than the information should look like that at the end: "Verbindung besteht direkt" or "Verbindung besteht indirekt (x)".

Does that make sense for you?

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 09:45 CEST, schrieb JRoberts ***@***.***>:

**Comment by mlanp** [comment]
Sorry, maybe wrong written or documented by me?

Pls leave the first letter for this word in uppercase. We do not write the word "gruppe" in German like that! It always has to be(!) "Gruppe" and this is absolutely right as status information and in all other use cases as well.

It is just lowercase if it is "attached" to some additional information like "Freundesgruppe" (group of friends). But I guess this is not the use case in this context?

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 09:57 CEST, schrieb JRoberts ***@***.***>:

**Comment by jr-simplex** [comment]
> I would change the translation in front of this status. At the moment I see in the German translation "Verbindung steht". I would change it to "Verbindung besteht". Than the information should look like that at the end: "Verbindung besteht direkt" or "Verbindung besteht indirekt (x)". Does that make sense for you?

Not sure, maybe it should be just "Verbindung" "direkt/indirekt"?

It looks like this in the UI:
image

**Comment by jr-simplex** [comment]
> It is just lowercase if it is "attached" to some additional information like "Freundesgruppe" (group of friends). But I guess this is not the use case in this context?

yep, it's not the case, let's leave uppercase then if the word itself is uppercase

**Comment by mlanp** [comment]
Yes, if you see it just as pure information / status about the connection this would be ok "Verbindung direkt/indirekt (x)".

If it should be a more formal information about the status of the connection I would write it as "Verbindung besteht direkt/indirekt".

-------------------------------------------------------------------------
Am Freitag, 28. Oktober 2022 10:14 CEST, schrieb JRoberts ***@***.***>:

**Comment by jr-simplex** [comment]
@mlanp is it good to review/merge?

**Comment by mlanp** [comment]
Hi,yes, from my point of view it is ok to merge it.Yesterday we discussed the different topics/questions from your site. For me the discussion is finished, as there are no further questions from you.Maybe one "open topic" regarding the connection information?Remember:"Yes, if you see it just as pure information / status about the connection this would be ok "Verbindung direkt/indirekt (x)".If it should be a more formal information about the status of the connection I would write it as "Verbindung besteht direkt/indirekt"."Depending on your decision I kindly ask you to use it as you think and correct my actual translation if needed.Thank you and merge it if it is ok for you.Am 29.10.2022 um 09:35 schrieb JRoberts ***@***.***>:﻿
@mlanp is it good to review/merge?

**Pull request merged**